app/ide-desktop/lib/content/src/confirmation.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
```

Comment:
this file name is enigmatic. What is `content/confirmation`? Is this a confirmation of content of the app?